packages/common/src/account/kms/kmsKeyToAddress.test.ts

```diff
@@ -102,4 +107,35 @@ describe("createKmsAccount", () => {
 
     expect(valid).toBeTruthy();
   });
+
+  it("can execute transactions", async () => {
+    const adminClient = createWalletClient({
+      chain: foundry,
+      transport: http(anvilRpcUrl),
+      account: privateKeyToAccount("0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"),
```

Comment:
wanna make a note here about this being a default anvil user?

can also use `testClient` to set balance of ~any account

Reply:
`testClient` would be more intuitive, the funding stuff is a means to an end